operator/pkg/apis/istio/v1alpha1/values_types.proto

```diff
@@ -602,7 +602,10 @@ message GlobalConfig {
   // Currently, two options are supported: "third-party-jwt" and "first-party-jwt".
   string jwtPolicy = 57;
 
-  // The next available key is 58
+  // Controls whether the in-cluster MTLS key and certs are loaded from the secret volume mounts.
+  google.protobuf.BoolValue SecretVolumeMtlsCertsEnabled = 4;
```

Comment:
nit - we normally don't have anything capitalized. also I think this needs to match what is in the templates (this one has Enabled suffix).

Maybe just change this to "mountCerts" or something a bit shorter?

Reply:
Done. Changed to "mountMtlsCerts" to specify it's incluster mTLS certs.